CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+#### Added
+- Added support for [Contentful Delivery API Search Parameters](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters)
+- Swift 3.1, Xcode 8.3 support.
+
+#### Changed
+- 
+
+#### Removed
+- 
+
+#### Fixed
+- 
+
+#### Deprecated
```

Comment:
Up to you but personally I only ad these sections the first time I need to use them for a given release. I don't trust myself to remember to remove them 😅